src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/DoNotGuardDictionaryRemoveByContainsKeyTests.cs

```diff
@@ -45,12 +45,11 @@ public class MyClass
 
         public MyClass()
         {
-            {|#0:MyDictionary.Remove(""Key"")|};
+            [|MyDictionary.Remove(""Key"")|];
```

Comment:
The `[| ... |]` syntax means that there is a diagnostic. If the fixed code doesn't have a diagnostic, don't add it.